main.go

```diff
@@ -113,32 +191,7 @@ func paConnectionWatchdog(ui *uistate) {
 		ui.paClient = paClient
 		go updateNoiseSupressorLoaded(paClient, &ui.noiseSupressorState)
```

Comment:
`UpdateNoiseSupressorLoaded` is to display the status in the top right, i think serves no purpose in a CLI version, we should probably avoid calling it.

Reply:
I deferred execution of `paConnectionWatchDog` until after the CLI-exclusive flags have been checked, so this should never be executed until we are certain that the watchdog is actually needed.